pyproject.toml

```
@@ -11,7 +11,7 @@ documentation = "https://docs.ephios.de/en/stable/"
 
 [tool.poetry.dependencies]
 python = "^3.8"
-django = "~3.2"
+django = ">=3.2,<4.1"
```

Should be 
```suggestion
django = ">=4.0,<4.1"
```
probably. @jeriox merge this if you agree.